.github/workflows/release.yml

```diff
@@ -61,6 +61,7 @@ jobs:
 
       - name: Compile Phoenix Assets
         working-directory: ${{env.working-directory}}
+        shell: bash
```

Comment:
Is this because the default shell for windows is powershell?

Reply:
Yep that's correct